### PR TITLE
Added maxmind.default.override parameter to CRConfig to handle maxmind default locations

### PR DESCRIPTION
**Pull request by rivasj**
Maintain a list of country code/coordinate pairs as: CountryCode;Lat,Long - in CRConfig.json to use instead of a default location for a given country. When a default location is indicated, TR checks if that country code has an entry specified in the config. If it does, it will set the client location to use the coordinates in that entry.

### CODE REVIEW

**Comment by asfgit** [comment]
Can one of the admins verify this patch?

File: docs/source/admin/traffic_ops/configuration.rst [hunk 1]
```diff
@@ -158,6 +158,9 @@ Many of the settings for the different servers in a Traffic Control CDN are cont
 +--------------------------+---------------+---------------------------------------------------------------------------------------------------------------------------------------+
 | geolocation6.polling.url | CRConfig.json | The location to get the IPv6 GeoLiteCity database from.                                                                               |
 +--------------------------+---------------+---------------------------------------------------------------------------------------------------------------------------------------+
+| maxmind.default.override | CRConfig.json | The destination geo coordinates to use for client location when maxmind returns a default location that matches the country code.     |
```
Comment (limited) [inline docs/source/admin/traffic_ops/configuration.rst]: Can we enter more than 1 country at a time? i.e. US and Canada?

File: traffic_router/core/src/main/java/com/comcast/cdn/traffic_control/traffic_router/core/router/TrafficRouter.java [hunk 2]
```diff
@@ -86,6 +87,8 @@
 	private final ConsistentHasher consistentHasher = new ConsistentHasher();
 	private SteeringRegistry steeringRegistry;
 
+	private final Map<String, Geolocation> defaultGeolocations = new HashMap<String, Geolocation>();
```
Comment (limited) [inline traffic_router/core/src/main/java/com/comcast/cdn/traffic_control/traffic_router/core/router/TrafficRouter.java]: nitpick, but variable might be more clear named defaultGeolocationsOverride. Skimming the code I got this confused with defaultLocation (which is what we call a geolocation from Maxmind w/o a city or postal code)

File: docs/source/admin/traffic_ops/configuration.rst (hunk 1 above)
Comment (elsloo) [reply docs/source/admin/traffic_ops/configuration.rst]: Yes, this parameter can be specified multiple times with different values. The documentation should be updated to reflect that.

@rivasj can you update the documentation to reflect this?

Comment (rivasj) [reply docs/source/admin/traffic_ops/configuration.rst]: I've updated documentation to show support for multiple parameter values.

File: traffic_router/core/src/main/java/com/comcast/cdn/traffic_control/traffic_router/core/router/TrafficRouter.java (hunk 2 above)
Comment (rivasj) [reply traffic_router/core/src/main/java/com/comcast/cdn/traffic_control/traffic_router/core/router/TrafficRouter.java]: I've renamed defaultGeolocations to defaultGeolocationsOverride

**Pull request merged**